noty-android/app/composeapp/src/main/java/dev/shreyaspatil/noty/composeapp/component/text/CommonTextField.kt

```diff
@@ -81,6 +82,7 @@ fun BasicNotyTextField(
             )
         }
         BasicTextField(
+            modifier = Modifier.fillMaxWidth(),
```

Comment:
@yogeshpaliyal Let this modifier be provided from the function parameter so that we can configure it as per our need and reuse it.

Reuse modifier of this function and make it like this

```kotlin
BasicTextField(
    modifier = modifier,
    decorationBox = { inlineField -> 
        Box { // Move outer box to decorationbox
            AnimatedVisibility(...) {}
            inlineField()
        }
    }
```

Reply:
okay

Reply:
@PatilShreyas  Added `textFieldModifier` param in **BasicNotyTextField**

Reply:
Okay